lib/airbrakex/exception_parser.ex

```diff
@@ -1,11 +1,11 @@
 defmodule Airbrakex.ExceptionParser do
   @moduledoc false
 
-  def parse(exception) do
+  def parse(exception, stacktrace) do
```

Comment:
`__STACKTRACE` can now only be used inside of `rescue` or `catch` clauses. That's why we need to receive it as an argument to this function.